### ISSUE TITLE
[SCREENREADER]: Inline error messages are announcing to screen readers on correction, not on subsequent errors

### ISSUE THREAD

**Issue by andreahewitt-odd** [department-of-veterans-affairs/va.gov-team#1063]
## Issue
Form inputs have inline error messages that are designed to announce themselves to screen readers when an input is invalid or empty. The error messages are firing correctly when inputs are blurred, but are announcing when users re-focus the input and provide a valid entry. The inputs are also not announcing the error message when users provide a second invalid entry.

## Steps to Reproduce
1. Go to any form input, especially a standard `input[type="text"]` which is where the behavior was noted
2. Focus, and blur the input, preferably using `TAB` or arrow keys
3. Verify the error message announces itself
4. Refocus the input
5. Verify the error message is not read aloud
6. Key in a valid entry
7. Verify the error message is read aloud, even though a valid entry was provided
8. Key in an invalid entry a second time
9. Blur and re-focus the input
10. Key in a second invalid entry
11. Verify the error message is not read aloud, even though the input was in error

## Related Issues
* https://github.com/usds/us-forms-system/issues/278

## Acceptance Criteria
The error message should be announced to screen readers when an input is blurred with invalid or empty data. The error message should fire a second time when invalid data is entered a second time, but should not announce when users enter valid data to correct the first error.

## Possible Fix
I worked up a simple solution on CodePen, after much testing locally: https://codepen.io/tpierce_402/pen/Lqezye

I discovered in testing that creating a `<span>` to hold the error message and adding and removing the `role="alert"` attribute would stop IE11 + JAWS from reading the error message a second time. 

Visual users could have the same experience where the error doesn't drop out until they start typing. Aural users will hear **Field name, invalid text entry**. The error message will fire again when the field is blurred if the input is invalid or empty.

Tested on:
* Win10 + IE11 + JAWS
* Win10 + Chrome + JAWS
* Win10 + Firefox + NVDA
* MacOS Mojave + Safari + VoiceOver

---

## Screenshots
<img width="532" alt="screen shot 2018-09-20 at 9 54 57 pm" src="https://user-images.githubusercontent.com/934879/45858462-23631080-bd23-11e8-945a-9acd620487ad.png">

---

<img width="529" alt="screen shot 2018-09-20 at 9 55 13 pm" src="https://user-images.githubusercontent.com/934879/45858468-2a8a1e80-bd23-11e8-9a7e-20e4e7255641.png">

**Comment by 1Copenut** [comment]
Closing this one as a duplicate to #314
